config/transformers/shadowToCss.ts

```diff
@@ -1,6 +1,6 @@
 import {toHex} from 'color2k'
-import StyleDictionary from 'style-dictionary'
-import {ShadowTokenValue} from '../../types/ShadowTokenValue'
+import type StyleDictionary from 'style-dictionary'
```

Comment:
Being a bit pedantic, but can you put the type imports after the main ES imports, rather than in-between?

Reply:
Hmm, generally I am fine with this. But can we do this over time? I don't want to sort 50 files now. 🙃 

Maybe we can add our own style guidelines? And add this? There is no eslint rule for this, right?